src/shared/bg_public.h

```diff
@@ -1968,5 +1956,24 @@ inline float Alignment2D(glm::vec3 u, glm::vec3 v, float z_factor = 0.0f)
 	return ScalarProduct2D(u, v, z_factor) / ( Length2D(u, z_factor) * Length2D(v, z_factor) );
 }
 
+inline float VectorNormalize(glm::vec3 &vec)
+{
+	float length2 = glm::length2(vec);
+
+	if (length2 != 0.0f)
+	{
+		vec *= Q_rsqrt(length2);
+	}
+
+	return length2;
+}
+
+inline void VectorCopy(const glm::vec3 &in, vec3_t out)
```

Comment:
The VectorCopy from q_shared.h already works for that case.

Reply:
Thanks, removed.